_posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md

```diff
@@ -13,24 +13,27 @@ The Pulp team is pleased to announce the release of the Pulp Ansible 0.2.0 plugi
 
 If you’re an Ansible user and do not want to host your private content on Ansible Galaxy, you can add the Pulp Ansible plugin to Pulp. You can then mirror the public Ansible content that you require and use Pulp as an on-premise platform to manage and distribute a scalable blend of public and private Ansible roles and collections across your organization.
 
-With the Pulp Ansible plugin, you can complete the following actions via the Pulp API. Ansible content includes both roles and collections:
+With the Pulp Ansible plugin, you can complete the following workflows via the Pulp API. Ansible content includes both roles and collections:
 
 * Mirror a subset of Ansible content on-premise
 * Mirror all of Ansible Galaxy’s content on-premise
-* Store private Ansible content on-premise
-* Install Ansible content from pulp_ansible using the ansible-galaxy CLI
-* Version Ansible content over time and rollback if necessary
-* Support for the new multi-role content type from Galaxy
+* Create a distribution and use the `ansible-galaxy` command to install content from this distribution
 
 For more information, see the [roles](https://pulp-ansible.readthedocs.io/en/latest/workflows/roles.html) and [collections](https://pulp-ansible.readthedocs.io/en/latest/workflows/collections.html) workflows in the Pulp Ansible documentation.
 
+The Pulp Ansible plugin also has the following features:
+
+* Store private Ansible content on-premise
```

Comment:
@bmbouter Do we support this via `ansible-galaxy push`?
I think that would be worth mentioning.

Reply:
Yes we support uploading using `ansible-galaxy collection publish`. The `ansible-galaxy role --help` doesn't have an upload capability in the client, so we only support the uploading of collections.

Reply:
There is no role upload/publish here: https://pulp-ansible.readthedocs.io/en/latest/workflows/roles.html
There are collection publish docs here: https://pulp-ansible.readthedocs.io/en/latest/workflows/collections.html